src/js/modules/components/team-modal.js

```diff
@@ -0,0 +1,17 @@
+import { refs } from './refs';
+const { aboutDev, teamModalBtnClose, backdrop } = refs;
+
+aboutDev.addEventListener('click', openTeamModal);
+teamModalBtnClose.addEventListener('click', closeTeamModal);
+window.addEventListener('click', closeTeamModal);
+window.addEventListener('keydown', closeTeamModal);
+
+function openTeamModal() {
+  backdrop.classList.remove('is-hidden');
+}
+
+function closeTeamModal(ev) {
+  if (ev.target === backdrop || ev.keyCode === 27 || ev.target === teamModalBtnClose) {
```

Comment:
What do these checks do?

Reply:
These conditions check that the window is closed by clicking on a button, on a backdrop or pressing the Esc button.